### PR TITLE
docs: first draft for assgiment text

### PR DESCRIPTION
**Pull request by havardh**

### CODE REVIEW

File: template-service/Readme.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  will use to reduce the number of operations it will do.
+  Examples of typical reductions includes:
+
+Avoid reinstall platform build dependencies on each build
+
+Avoid reinstall code dependencies when only code has changed.
+
+#### Create the Kubernetes deployment configuration
+
+These steps should be done in [service.yml](./service.yml)
```
Comment (havardh) [inline template-service/Readme.md]: @larseen: Feel free to make suggestions about better descriptions and more optional advanced tasks here ✌️

File: template-service/Readme.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+```
+[
+  {
+    x: 5,
+    y: 5,
+    dx: 20,
+    dy: 30,
+    dz: 6
+  }
```
Comment (hawkaa) [inline template-service/Readme.md]: I think it's a little unclear that there can be multiple buildings. Perhaps add another building to the example or add a comment below?

File: template-service/Readme.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  and [services-service](../platform-services/services-service/Dockerfile) for examples.
+
+(there is a small prize for the smallest working image)
+
+- (optional) g) Make your Docker build run the minimal number of step for each
+  subsequent builds. Your local installation of Docker includes a cache which it
+  will use to reduce the number of operations it will do.
+  Examples of typical reductions includes:
+
+Avoid reinstall platform build dependencies on each build
```
Comment (hawkaa) [inline template-service/Readme.md]: These should probably be bullets?

File: template-service/Readme.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+- c) Add a Service `kind: Service`
+
+- d) Add an Ingress `kind: Ingress`
+
+- (optional) e) Configure the resource utilization appropriate for your langauge.
+
+#### Deploy the service to Kubernetes
+
+- a) Tag your docker image. `docker tag <image-name> <username>:<image-name>`
```
Comment (hawkaa) [inline template-service/Readme.md]: I originally added this as a part of the docker setup. The rationale is that dockerhub is a powerful platform on its own, and I don't want it mixed up with Kubernetes. What do you think?

File: template-service/Readme.md (hunk 3 above)
Comment (havardh) [reply template-service/Readme.md]: Yes, I didn't figure out the proper overall layout 😅

File: template-service/Readme.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+(This means create a 30x20 meters building with a height of 9 meters and place
+it 5 meters from the top and left edges)
+
+You can test your server using `$ curl localhost:8080`, assuming that your server
+runs on port 8080. It should reply with the JSON structure above.
+
+- (optional) d) Place the buildings in a fixed, but personalized layout.
+
+- (optional) e) Write an algorithm for optimizing the placement of the buildings.
```
[the remark this answers is not in the document]
Comment (havardh) [reply template-service/Readme.md]: It has now.

**Pull request merged**